### PR TITLE
[INTERNAL] CI: Adds .NET8 SDK support

### PR DESCRIPTION
**Pull request by kirankumarkolli**
[INTERNAL] CI: Adds .NET8 SDK support

RISK: not all pipelines are exercised part of CI gates and might be surprises later.

### CODE REVIEW

File: azure-pipelines-encryption-custom.yml [hunk 1]
```diff
@@ -21,7 +21,13 @@ stages:
         displayName: Use .NET 6.0
         inputs:
           packageType: 'sdk'
```
Comment (Pilchie) [inline azure-pipelines-encryption-custom.yml:23]: Could potentially change this to a .NET 6 runtime instead.

**Review by Pilchie — COMMENTED** [review]
Do we want to actually run the tests with both, instead of just building with the .NET 8 SDK?

**Comment by kirankumarkolli** [comment]
> Do we want to actually run the tests with both, instead of just building with the .NET 8 SDK?
That's next follow-up PR.

**Review by NaluTripician — APPROVED** [review]
LGTM

**Review by Pilchie — APPROVED** [review]

**Pull request merged**